packages/manager/apps/dedicated-servers/src/components/monitoringStatus.tsx

```diff
@@ -6,15 +6,20 @@ import { DedicatedServer } from '@/data/types/server.type';
 
 export const MonitoringStatusChip = (server: DedicatedServer) => {
   const { t } = useTranslation('dedicated-servers');
-  const color = !server.monitoring
-    ? ODS_BADGE_COLOR.warning
-    : ODS_BADGE_COLOR.success;
+
   const getMonitoringStatusLabel = (
     _monitoring: boolean,
     _noIntervention: boolean,
   ) => {
     if (!_monitoring) return 'disabled';
-    return _noIntervention ? 'proactive' : 'no-proactive';
+    return _noIntervention ? 'no-proactive' : 'proactive';
+  };
+  const getMonitoringStatusColor = (
+    _monitoring: boolean,
+    _noIntervention: boolean,
+  ) => {
+    if (!_monitoring) return ODS_BADGE_COLOR.warning;
+    return _noIntervention ? ODS_BADGE_COLOR.warning : ODS_BADGE_COLOR.success;
   };
```

Comment:
I suggest to export function outside the component. This avoid to redeclare during each re-rending. And it's easier to test without use a component render.

And rename props of function to respect nomenclature of boolean
And use "if () return" for each case allow better readibility

```suggestion
const getMonitoringStatusLabel = (
isMonitoring: boolean,
hasNoIntervention: boolean,
) => {
    if (!isMonitoring) return 'disabled';
    if (hasNoIntervention) return 'no-proactive';
    return 'proactive';
  };
  const getMonitoringStatusColor = (
    isMonitoring: boolean,
    hasNoIntervention: boolean,
  ) => {
    if (!isMonitoring) return ODS_BADGE_COLOR.warning;
    if (hasNoIntervention) return ODS_BADGE_COLOR.warning;
    return ODS_BADGE_COLOR.success;
};

export const MonitoringStatusChip = (server: DedicatedServer) => {
const { t } = useTranslation('dedicated-servers');


```

Reply:
Done :)